test/recipes/80-test_ssl_new.t

```diff
@@ -78,7 +78,7 @@ my %skip = (
   "15-certstatus.conf" => $no_tls || $no_ocsp,
   "16-dtls-certstatus.conf" => $no_dtls || $no_ocsp,
   "18-dtls-renegotiate.conf" => $no_dtls,
-  "19-mac-then-encrypt.conf" => $no_pre_tls1_3
+  "19-mac-then-encrypt.conf" => $no_pre_tls1_3 || disabled("tls1_2")
```

Comment:
I haven't quite understood why, I just know that it fails if TLSv1.2 is disabled (`no-tls1_2`).  Maybe you have an explanation?